de.fu_berlin.inf.dpp/src/de/fu_berlin/inf/dpp/StartupSaros.java

```diff
@@ -63,7 +63,9 @@ public void earlyStartup() {
      * needed by the Feedback component we have to initialize them here
      */
 
-    FeedbackPreferences.applyDefaults(preferenceStore);
+    if (Boolean.getBoolean("de.fu_berlin.inf.dpp.feedback.ENABLED")) {
```

Comment:
This is not really part of the feature. It only fills some property stuff and is harmless if executed.

Reply:
Ok, I will remove the superfluous check